packages/xml-views-completion/test/api-spec.ts

```diff
@@ -104,23 +106,335 @@ describe("The `getXMLViewCompletions()` api", () => {
       });
     });
   });
+
+  context("filter deprecated items according to settings", () => {
+    const NO_DEPRECATED_SUGGESTIONS = {
+      codeAssist: { deprecated: false, experimental: true },
+    };
+    const ALLOW_DEPRECATED_SUGGESTIONS = {
+      codeAssist: { deprecated: true, experimental: true },
+    };
+
+    function testDeprecated(
```

Comment:
could should we combine `testDeprecated` and `testExperimental` by passing settings argument to reduce LOC ?

Reply:
Done